### PR TITLE
Porting over Kryson's carpets.

### PR DESCRIPTION
**Pull request by PetMudstone**
## About The Pull Request

Ports over Kryson's carpets from https://github.com/tgstation/tgstation/pull/54612. Does not add the spawners nor any way to obtain them at the moment. 
Below images copied from relevant PR.
Executive NT carpet.
image
Stellar carpet.
image
Donk Co. promotional carpet.
image

## Why It's Good For The Game
Carpets look pretty. Could be used later by mappers or by players. Also, Donk carpets are *integral* to Donk Co. RP.

## Changelog
:cl: Pet Mudstone, Kryson
add: Ported over some of Kryson's special carpets: Donk Co., stellar, and executive NT.
/:cl:

### CODE REVIEW

File: code/game/turfs/open/floor/fancy_floor.dm [hunk 1]
```diff
@@ -289,6 +289,33 @@
 	floor_tile = /obj/item/stack/tile/carpet/royalblue
 	smoothing_groups = list(SMOOTH_GROUP_TURF_OPEN, SMOOTH_GROUP_CARPET_ROYAL_BLUE)
 	canSmoothWith = list(SMOOTH_GROUP_CARPET_ROYAL_BLUE)
+
```
Comment (martinlyra) [inline code/game/turfs/open/floor/fancy_floor.dm]:
```suggestion

```

Superflous whitespace here. And that is it.

**Review by martinlyra — CHANGES_REQUESTED** [review]

File: code/game/turfs/open/floor/fancy_floor.dm [hunk 2]
```diff
@@ -289,6 +289,32 @@
 	floor_tile = /obj/item/stack/tile/carpet/royalblue
 	smoothing_groups = list(SMOOTH_GROUP_TURF_OPEN, SMOOTH_GROUP_CARPET_ROYAL_BLUE)
 	canSmoothWith = list(SMOOTH_GROUP_CARPET_ROYAL_BLUE)
+/turf/open/floor/carpet/executive
```
Comment (martinlyra) [inline code/game/turfs/open/floor/fancy_floor.dm:293]:
```suggestion

/turf/open/floor/carpet/executive
```

Try this

**Comment by ZephyrTFA** [comment]
Check your line endings. Make sure they're consistent maybe?

**Comment by martinlyra** [comment]
After #163 gets merged, run the CI again on this to get the green ticks. It is not done automatically whenever a PR has been merged for performance reasons. So just push an empty commit or trigger the CI manually to restart it.

Sorry for the inconvenience!

**Review by martinlyra — APPROVED** [review]
Should be fixed! All you need to do now is to push a commit of whatever to restart the CI.

**Pull request merged**